src/services/maci/constants.ts

```diff
@@ -1,39 +1,40 @@
 import {
-    EContracts,
-    PollFactory__factory as PollFactoryFactory,
-    MessageProcessorFactory__factory as MessageProcessorFactoryFactory,
-    TallyFactory__factory as TallyFactoryFactory,
-    MACI__factory as MACIFactory,
-    ConstantInitialVoiceCreditProxy__factory as ConstantInitialVoiceCreditProxyFactory,
-    EASGatekeeper__factory as EASGatekeeperFactory,
-    Verifier__factory as VerifierFactory,
-    PoseidonT3__factory as PoseidonT3Factory,
-    PoseidonT4__factory as PoseidonT4Factory,
-    PoseidonT5__factory as PoseidonT5Factory,
-    PoseidonT6__factory as PoseidonT6Factory,
-    VkRegistry__factory as VkRegistryFactory,
-  } from "maci-cli/sdk";
+  PollFactory__factory as PollFactoryFactory,
+  MessageProcessorFactory__factory as MessageProcessorFactoryFactory,
+  TallyFactory__factory as TallyFactoryFactory,
+  MACI__factory as MACIFactory,
+  ConstantInitialVoiceCreditProxy__factory as ConstantInitialVoiceCreditProxyFactory,
+  EASGatekeeper__factory as EASGatekeeperFactory,
+  Verifier__factory as VerifierFactory,
+  VkRegistry__factory as VkRegistryFactory,
+} from "maci-cli/sdk";
 import type { IAbi } from "./types";
 
 /**
  * Constant variables
  */
 export const STATE_TREE_SUB_DEPTH = 2;
+export const STATE_TREE_DEPTH = 10;
```

Comment:
I don't think these should be constants but should be customizable

Reply:
already open another issue for this: https://github.com/privacy-scaling-explorations/maci-rpgf/issues/112